client/src/Molecules/Footer/FooterDesktop.js

```diff
@@ -0,0 +1,86 @@
+import FooterDesktopInnerStyled from './FooterDesktopInnerStyled';
+import FooterDesktopStyled from './FooterDesktopStyled';
+import logoDevsOnTheWaves from '../../Assets/logoDevsOnTheWaves.svg';
+import { Copy, DevsLogo } from './FooterStyled';
+import { LogoStyled } from './FooterStyled';
+
+const FooterDesktop = () => {
+  return (
+    <>
+      <FooterDesktopStyled>
+        <FooterDesktopInnerStyled>
+          <a
+            href="https://github.com/CC2021-WBL"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <LogoStyled>
+              <DevsLogo>
+                <img
+                  className="logo"
+                  src={logoDevsOnTheWaves}
+                  alt="logo"
+                  width="35px"
+                />
+              </DevsLogo>
+              <Copy>
+                Copyright <br />
+                #Devs on the Waves
+              </Copy>
+            </LogoStyled>
+          </a>
+          <div>
+            <a
+              href="https://github.com/CC2021-WBL"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <h3>O aplikacji</h3>
+            </a>
+            <br></br>
+            <a
+              href="https://github.com/CC2021-WBL"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <p>Czym jest GOOD GOI APP</p>
```

Comment:
I think it deosnt need to be enclosed in a parapgrah tag, its just text.

Reply:
Paragraph has just the right font size, weight and color as per Figma design. It will be less code to use p after all.